test/cli.py

```diff
@@ -232,7 +232,6 @@ def setUp(self) -> None:
 
         return super().setUp()
 
-    """
```

Comment:
That is so hard to notice in code review 😂 but has such an impact on the code!

Reply:
yeap!